core/include/ldml/keyboardprocessor_ldml.ts

```diff
@@ -245,6 +245,80 @@ class Constants {
    */
   readonly keys_flags_extend = 1;
 
+  /**
+   * Constant for no modifiers
+   */
+  readonly keys_mod_none = 0;
+  /**
+   * bitmask for modifier key
+   */
+  readonly keys_mod_altL = 1 << 2;
+  /**
+   * bitmask for modifier key
+   */
+  readonly keys_mod_altR = 1 << 3;
+  /**
+   * bitmask for either alt
+   */
+  readonly keys_mod_alt  = this.keys_mod_altL | this.keys_mod_altR;
+  /**
+   * bitmask for modifier key
+   */
+  readonly keys_mod_caps = 1 << 7;
+  /**
+   * bitmask for modifier key
+   */
+  readonly keys_mod_cmd =  1 << 15;
+  /**
+   * bitmask for modifier key
+   */
+  readonly keys_mod_ctrlL = 1 << 0;
+  /**
+   * bitmask for modifier key
+   */
+  readonly keys_mod_ctrlR = 1 << 1;
+  /**
+   * bitmask for either ctrl
+   */
+  readonly keys_mod_ctrl = this.keys_mod_ctrlL | this.keys_mod_ctrlR;
+  /**
+   * bitmask for modifier key
```

Comment:
```suggestion
   * bitmask for Option modifier key
```

Do we need this? Alt and Option are synonymous.